__main__.py

```diff
@@ -173,6 +172,11 @@ def main():
             debug = False
             sys.tracebacklimit = 0
 
+        if hasattr(args, 'not_cached') and args.not_cached:
```

Comment:
I don't think so the ```hasattr(args, 'not_cached')``` is required.
Though it has been added while checking for the ```--debug``` argument in ```__main__.py```